tests/cloud-http-get-test.yaml

```diff
@@ -0,0 +1,12 @@
+meta:
+  version: 0.1.0
```

Comment:
I think that it would be good if we use the same version of https://github.com/infinyon/http-connector/blob/49074e3a24742243dcd52d1896bacf1fd62c6152/crates/http-source/hub/package-meta.yaml#L4

Maybe we can use sed to replace this value just like we do with topic name and connector name?

Reply:
Fixed here! [f538c98](https://github.com/infinyon/http-source-connector/pull/31/commits/f538c9891924f17d0dbb31709ffce04509336e6d)